packages/lesswrong/components/tagging/history/TagHistoryPage.tsx

```diff
@@ -31,16 +70,31 @@ const TagHistoryPage = ({classes}: {
   }
   
   return <SingleColumnSection>
-    <Link to={tagGetUrl(tag)}><h1 className={classes.title}>{tag.name}</h1></Link>
+    <SectionTitle title={tag.name} href={tagGetUrl(tag)}>
+      <div onClick={ev => setSettingsExpanded(expanded => !expanded)}>
+        <Components.SettingsButton label="Settings" />
+      </div>
+    </SectionTitle>
+    
+    <TagHistoryFeedSettings
```

Comment:
FYI: this is a new addition to the page which seems like the EA forum probably wants, to the extent that anyone is using the tag history page at all.  It lets users filter what type of history items they see on the page.